presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/MultipleInputQuestionStepFragment.kt

```diff
@@ -0,0 +1,71 @@
+package org.sagebionetworks.assessmentmodel.presentation
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import org.sagebionetworks.assessmentmodel.Step
+import org.sagebionetworks.assessmentmodel.presentation.databinding.FragmentMultipleInputQuestionStepBinding
+import org.sagebionetworks.assessmentmodel.serialization.ChoiceQuestionObject
+import org.sagebionetworks.assessmentmodel.serialization.SimpleQuestionObject
+import org.sagebionetworks.assessmentmodel.survey.*
+
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [MultipleInputQuestionStepFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
```

Comment:
Would be good to have a comment documenting that for now this only works for a MultipleInputQuestion containing exactly three textfields.